pkg/golinters/gofmt_common.go

```diff
@@ -207,7 +207,29 @@ func (p *hunkChangesParser) parse(h *diffpkg.Hunk) []Change {
 	return p.ret
 }
 
-func extractIssuesFromPatch(patch string, log logutils.Log, lintCtx *linter.Context, isGoimports bool) ([]result.Issue, error) {
+func getErrorTextForLinter(lintCtx *linter.Context, linterName string) string {
+	text := "File is not formatted"
+	switch linterName {
+	case gofumptName:
+		text = "File is not `gofumpt`-ed"
```

Comment:
"File is not `gofumpt` -ed"

Reply:
Same below

Reply:
it is same style as it was: https://github.com/golangci/golangci-lint/pull/1239/files/79726260904207cbef41c72458a1d8d415d47f80#diff-4d48c424c434d446b09abeaa31ab7e91L230

Let's keep it in this way, we cannot change the error messages for the `gofmt` and `goimports` because it's backward incompatible changes. Maybe we can do it in `v2`, but not now